docs/src/pages/components/text-fields/text-fields.md

```diff
@@ -220,3 +220,4 @@ For more advanced use cases you might be able to take advantage of:
 - [formik-material-ui](https://github.com/stackworx/formik-material-ui) Bindings for using Material-UI with [formik](https://jaredpalmer.com/formik).
 - [redux-form-material-ui](https://github.com/erikras/redux-form-material-ui) Bindings for using Material-UI with [Redux Form](https://redux-form.com/).
 - [mui-rff](https://github.com/lookfirst/mui-rff) Bindings for using Material-UI with [React Final Form](https://final-form.org/react).
+- [react-hook-form](https://github.com/react-hook-form/react-hook-form) Bindings for using Material-UI with [React Hook Form](https://react-hook-form.com/).
```

Comment:
Where are the bindings?

Reply:
I'm wrong, I probably shouldn't have called it "bindings". React Hook Form is able to leverage the inputRef to work (it's sort of surprised me at how easy it was)

```jsx
import React from "react";
import { useForm } from "react-hook-form";

export default function App() {
  const { register, handleSubmit, watch, errors } = useForm();
  const onSubmit = data => console.log(data);

  console.log(watch("example")); // watch input value by passing the name of it

  return (
    {/* "handleSubmit" will validate your inputs before invoking "onSubmit" */}
    <form onSubmit={handleSubmit(onSubmit)}>
    	{/* register your input into the hook by invoking the "register" function */}
      <TextField
          variant="outlined"
          fullWidth
          inputRef={register}
          name={'name'}
          defaultValue={supplement.name}
          label="Supplement Name"
        />      
    </form>
  );
}
```

I'm not quite sure the best language to describe this ... lol